extensions/mssql/package.json

```diff
@@ -692,15 +701,20 @@
           "when": "connectionProvider == MSSQL && nodeType == Folder && objectType == Databases && !isCloud && config.workbench.enablePreviewFeatures",
           "group": "1_objectManagement@2"
         },
+        {
+          "command": "mssql.backupDatabase",
+          "when": "connectionProvider == MSSQL && nodeType == Database && !isCloud && mssql:engineedition != 11 && config.workbench.enablePreviewFeatures && (productQualityType =~ /^(insider|dev)$/ || isDevelopment)",
```

Comment:
The new entry has the preview tag on it to distinguish them

Reply:
Ok - and is that work to reach parity + remove old one planned to happen before enabling this for stable? Or were you thinking of having both in stable for at least 1 release?

Reply:
Since they're using the same APIs and backup options, having both in stable at the same time seems unnecessary. They're such similar experiences that it would be redundant (and even confusing) to have both exist side by side in stable

Reply:
Yup, I fully agree 😁